internal/pkg/agent/cmd/enroll_cmd_test.go

```diff
@@ -522,21 +617,55 @@ func TestValidateEnrollFlags(t *testing.T) {
 	t.Run("no flags", func(t *testing.T) {
 		cmd := newEnrollCommandWithArgs([]string{}, streams)
 		err := validateEnrollFlags(cmd)
-		require.NoError(t, err)
+
+		assert.NoError(t, err)
 	})
 
 	t.Run("service_token and a service_token_path are mutually exclusive", func(t *testing.T) {
+		absPath, err := filepath.Abs("/path/to/token")
+		require.NoError(t, err, "could not get absolute absPath")
```

Comment:
I honestly do not know why it was working on windows before, but it started to fail on `elastic-agent-cert-key does not require key-passphrase`, so I fixed all tests.